### PR TITLE
Rework introduction chapter

### PR DESCRIPTION
**Pull request by misappi**
I (hopefully) refined the introduction chapter. Changed and streamlined some formulation, unified usage of terminology and abbreviations etc.

TBD:
* Do we want to target only companies or "organizations" in general?
* Do we want to put little bit more meat to the list of motivations?

### CODE REVIEW

File: Introduction.md [hunk 1]
```diff
@@ -2,60 +2,59 @@
...
-2. Participation - Engagement with open source community
+2. Participation - Engagement with open source communities
 3. Contribution - Pragmatic contributions to open source projects
 4. Leadership - Strategic involvement with open source to drive business value
 
-To advance from one level to another it requires certain initiatives and structural and organizational elements.
+To advance from one level to another, certain initiatives and structural and organizational elements are required.
 
-Going from consumption to participation will start with informal engagement and low-effort activities such as reporting bugs in upstream projects driven by technical needs. Decisions about open source contributions will be ad-hoc and be taken for individual cases.
+Going from consumption to participation, for example, will start with informal engagement and low-effort activities such as reporting bugs in upstream projects, which typically is driven by technical needs. On thet level, decisions about open source contributions will normally be ad-hoc and be taken for individual cases only.
```
Comment (cornelius) [inline Introduction.md:21]: Small typo:

```suggestion
Going from consumption to participation, for example, will start with informal engagement and low-effort activities such as reporting bugs in upstream projects, which typically is driven by technical needs. On that level, decisions about open source contributions will normally be ad-hoc and be taken for individual cases only.
```

I'm also not sure, if the "for example" is the right term here. We say for each level how to advance to the next one, so it's not really an example, but one part of the complete list.

File: Introduction.md [hunk 2]
```diff
@@ -2,60 +2,59 @@
...
+An increasing number of organizations realized the tasks of managing open source in an enterprise and complex relationships that are inherent to the open source ecosystem when they are advancing in their engagement in open source. For this reason, many of them invest start Open Source Program Offices (OSPOs), something called differently, for example Open Source Technology Centers, Open Source Community Development Team etc. OSPOs are a designated place where open source is supported, nurtured, shared, explained, and grown inside an organization. With such an office in place, businesses can establish and execute on their open source strategies in clear terms and responsibilities, giving their leaders, developers, marketers, and other staff the processes and tools they need to make open source a success within their operations.
```
Comment (cornelius) [inline Introduction.md:31]: Either "inverst in" or "start". At the moment there is both.

**Review by cornelius — COMMENTED** [review]
The changes look good to me. There are two tiny comments inline.

Regarding the questions:

I would find it natural if we talk about companies, not organizations in general, because that's the perspective of us as the current group of authors. Many things are still valid for other types of organizations but there might be other factors we don't cover yet.

Giving some more details to the motivation why to do outbound open source would be nice. It would be ideal, if people who ask why they should do open source contributions, could be pointed to this section, and it would provide a convincing answer. On the other hand it already is a solid list of good reasons, so I would be also ok with keeping it as it is.

File: Introduction.md [hunk 3]
```diff
@@ -2,60 +2,76 @@
...
+
+* Integrating "own" features into upstream projects makes maintenance a lot easier
+* Upstream versions can be directly used in own products and services
+* More features are obtained in shorter period of time
+* Higher quality is achieved in shorter period of time
+* Support available from core experts
+
+### Exercise strategic influence
+
+Despite the benefits of open source software wrt. development velocity and quality mentioned above, contributing to open source projects can also be important from a strategic point of view. In open source world, reputation and the ability to influence is typically build up by engaging in the community and by contributing. Thus, contributions to open source projects can help to ...
```
Comment (cornelius) [inline Introduction.md:51]: Some small language suggestions:

```suggestion
In addition the benefits of open source software wrt. development velocity and quality mentioned above, contributing to open source projects can also be important from a strategic point of view. In the open source world, reputation and the ability to influence is typically build up by engaging in the community and by contributing. Thus, contributions to open source projects can help to ...
```

File: Introduction.md [hunk 4]
```diff
@@ -2,60 +2,76 @@
...
+
+Open source communities (particulary those run by the big open source foundations) provide a neutral place for collaboration between companies and other organizations. Thus, an open source approach could offer new ways of collaboration with suppliers, customers, partner and even competitors, just to mention industry- or domain-specific projects such as [Linux Foundation Energy](https://www.lfenergy.org/) or [Eclipse Tractus-X](https://projects.eclipse.org/proposals/eclipse-tractus-x). Establishing open source communities can also be a powerful means to create and maintain ecosystems and to establish defacto standards.
+
+### Attract, grow and retain talents
+
+Software (and therefore also open source software) becomes more and more ubiqitous in many products and areas. Thus, for many companies (we are not only referring to software or cloud companies, but also to companies from other segments, such as traditional hardware producers who integrate software into their products more and more) it is crucial to have a skilled and motivated software development workforce. An open source strategy including open source contributions and community engagements support this:
```
Comment (cornelius) [inline Introduction.md:61]: I would take this even wider and include all companies who are becoming dependent on software:

```suggestion
Software (and therefore also open source software) becomes more and more ubiqitous in many products and areas. Thus, for many companies (we are not only referring to software or cloud companies, but also to companies from other segments, such as traditional hardware producers who integrate software into their products more and more, or any other company which is becoming more dependent on software due to accelerating digital transformation) it is crucial to have a skilled and motivated software development workforce. An open source strategy including open source contributions and community engagements support this:
```

**Review by cornelius — COMMENTED** [review]
Some suggestions in separate comments. I like the new section.

File: Introduction.md [hunk 5]
```diff
@@ -2,60 +2,76 @@
...
+
+Despite the benefits of open source software wrt. development velocity and quality mentioned above, contributing to open source projects can also be important from a strategic point of view. In open source world, reputation and the ability to influence is typically build up by engaging in the community and by contributing. Thus, contributions to open source projects can help to ...
+
+* influence the direction of upstream open source projects
+* gain (co)copyright on open source software packages
+* access to the creativity of everyone interested in software
+
+Open source communities (particulary those run by the big open source foundations) provide a neutral place for collaboration between companies and other organizations. Thus, an open source approach could offer new ways of collaboration with suppliers, customers, partner and even competitors, just to mention industry- or domain-specific projects such as [Linux Foundation Energy](https://www.lfenergy.org/) or [Eclipse Tractus-X](https://projects.eclipse.org/proposals/eclipse-tractus-x). Establishing open source communities can also be a powerful means to create and maintain ecosystems and to establish defacto standards.
+
```
[the remark this answers is not in the document]
Comment (misappi) [reply Introduction.md:60]: I added that. Thanks.

**Pull request merged**